### PR TITLE
Add switch to disable building MMX

### PR DESCRIPTION
**Pull request by lorenzopivetta**
Add cmake switch to disable building MMX support. Targets 9.3-backports.

### CODE REVIEW

File: cppapi/server/jpeg_mmx/CMakeLists.txt [hunk 1]
```diff
@@ -20,21 +20,29 @@ if(WIN32)
...
         target_compile_options(jpeg_mmx_objects_dyn PRIVATE -O0)
-        target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE JPG_USE_ASM)
-        target_compile_options(jpeg_mmx_objects_sta PRIVATE -O0)
-        target_compile_definitions(jpeg_mmx_objects_sta PRIVATE JPG_USE_ASM)
+	target_compile_options(jpeg_mmx_objects_sta PRIVATE -O0)
+	if(TANGO_JPEG_MMX)
+            target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE JPG_USE_ASM)
+            target_compile_definitions(jpeg_mmx_objects_sta PRIVATE JPG_USE_ASM)
+	endif()
     endif()
```
Comment (mliszcz) [inline cppapi/server/jpeg_mmx/CMakeLists.txt]: Indentation is very inconsistent here. In just 10 lines of code we indent blocks by two, four and six spaces.
This is not a major issue, but please consider enabling auto-indentation in your editor for future PRs.

Comment (t-b) [reply cppapi/server/jpeg_mmx/CMakeLists.txt]: @mliszcz You are right. I think we should also have some rough coding guidelines for CMAKE. There is even https://github.com/cheshirekow/cmake_format although.

**Comment by t-b** [comment]
@lorenzopivetta Can you provide a PR with the same changes targetting tango-9-lts branch?

File: cppapi/server/jpeg_mmx/CMakeLists.txt (hunk 1 above)
Comment (lorenzopivetta) [reply cppapi/server/jpeg_mmx/CMakeLists.txt]: Agree, 6 spaces is clearly a misprint.  How many do we want? 2? 4? 8?

**Pull request merged**